### PR TITLE
Inject Youtube tip button

### LINKED ISSUE #429: Inject Youtube tip button
Similar functionality to the twitter tip button for youtube.

### PR DESCRIPTION
**Pull request by etharner**
Closes #429.

### CODE REVIEW

File: src/content-scripts/tipButton.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    e.stopPropagation();
+  };
+
+  buttonContent.onmouseenter = e => {
+    hoverEnter();
+    e.stopPropagation();
+  };
+  buttonContent.onmouseleave = e => {
+    hoverLeave();
+    e.stopPropagation();
```
Comment (davidyuk) [inline src/content-scripts/tipButton.js:82]: @mradkov why do we need `stopPropagation` here?

Comment (davidyuk) [reply src/content-scripts/tipButton.js:82]: In general, I would suggest to define hover styles using css.

File: src/content-scripts/tipButton.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      position: relative;
+    }
+
+    &:hover {
+      span {
+        color: #2a9cff;
+      }
+
+      img {
+        filter: invert(52%) sepia(46%) saturate(3166%) hue-rotate(186deg) brightness(100%) contrast(104%);
```
Comment (etharner) [inline src/content-scripts/tipButton.scss:43]: I propose to use that hack (https://codepen.io/sosuke/pen/Pjoqqp) instead of loading other icon

**Review by mradkov — CHANGES_REQUESTED** [review]
The button I see on YT does not have the appropriate style.

Screenshot 2020-08-20 at 10 40 03

File: src/content-scripts/tipButton.js [hunk 3]
```diff
@@ -1,8 +1,13 @@
 import icon from '../icons/twitter-icon-tip.svg';
-import './tipButton.scss';
 
 global.browser = require('webextension-polyfill');
 
+const style = document.createElement('link');
```
Comment (etharner) [inline src/content-scripts/tipButton.js:5]: I used solution proposed here [https://stackoverflow.com/a/9721569]()

**Review by mradkov — APPROVED** [review]
Looks good now, thanks @etharner.

**Pull request merged**